Debugging/NodeJs.md

```diff
@@ -0,0 +1,29 @@
+
+# NodeJs
+
+Installation instructions of the Tessel CLI can be found here: http://tessel.github.io/t2-start/
+
+This document will address common problems related to NodeJs usage.
+
+## NVM
+
+The official installation instructions assume Node is installed globally. For those of you using NVM this will not be the case. As indicated [here](https://github.com/creationix/nvm/issues/786) NVM does not install globally by design.
+
+### Linux
+
+Install `t2-cli` module as normal. To install the drivers, perform the following:
```

Comment:
> ... but udev rules that are installed under /etc/ to set the permissions so that your normal user account can access the device. This can only be installed as root.

I wonder if the script itself can request sudo access after it's run?